src/test/java/org/folio/rest/impl/MockServer.java

```diff
@@ -483,6 +483,10 @@ private void handleGetInstanceBulkIds(RoutingContext ctx) {
         bulkIds = new JsonObject(RestVerticleTestBase.getMockData(INSTANCE_BULK_IDS_ALL_VALID_MOCK_DATA_PATH));
         addServerRqRsData(HttpMethod.GET, RECORD_BULK_IDS, bulkIds);
         serverResponse(ctx, 200, APPLICATION_JSON, bulkIds.encodePrettily());
+      } else if (ctx.request().getParam("query").contains("(languages=\"uk\")")) {
+        bulkIds = new JsonObject(RestVerticleTestBase.getMockData(INSTANCE_BULK_IDS_WITH_RANDOM));
```

Comment:
We can move these lines into one method, since they are repeated three times:
` bulkIds = new JsonObject(RestVerticleTestBase.getMockData(INSTANCE_BULK_IDS_MOCK_DATA_PATH));`
`addServerRqRsData(HttpMethod.GET, RECORD_BULK_IDS, bulkIds);`
`serverResponse(ctx, 200, APPLICATION_JSON, bulkIds.encodePrettily());`

Reply:
updated. thanks